From: "Barak A. Pearlmutter" <bap@debian.org>
Date: Fri, 4 Sep 2026 12:00:00 +0000
Subject: info dircategory
Bug-Debian: https://bugs.debian.org/115450
Forwarded: not-needed

File the info manuals under "The Algorithmic Language Scheme"

The MIT/GNU Scheme manuals were by themselves under "Programming
Languages" in the top-level Info directory, while other Debian Scheme
systems (Guile, SCM, SLIB, XlibScm) all use section "The Algorithmic
Language Scheme".

Use the same @dircategory as everybody else, in all seven manuals (the
four in doc/ plus the three plugin manuals under src/), so that they
are listed together, and update doc/info-dir -- the fallback top-level
Info node, installed only when none exists -- to match.
---
 doc/ffi/mit-scheme-ffi.texi           | 2 +-
 doc/info-dir                          | 2 +-
 doc/ref-manual/mit-scheme-ref.texi    | 2 +-
 doc/sos/mit-scheme-sos.texi           | 2 +-
 doc/user-manual/mit-scheme-user.texi  | 2 +-
 src/blowfish/mit-scheme-blowfish.texi | 2 +-
 src/gdbm/mit-scheme-gdbm.texi         | 2 +-
 src/imail/mit-scheme-imail.texi       | 2 +-
 8 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/doc/ffi/mit-scheme-ffi.texi b/doc/ffi/mit-scheme-ffi.texi
index 1a08be3..92564a1 100644
--- a/doc/ffi/mit-scheme-ffi.texi
+++ b/doc/ffi/mit-scheme-ffi.texi
@@ -35,7 +35,7 @@ Documentation License.''
 @end quotation
 @end copying
 
-@dircategory Programming Languages
+@dircategory The Algorithmic Language Scheme
 @direntry
 * MIT/GNU Scheme FFI: (mit-scheme-ffi).
                                 Foreign Function Interface
diff --git a/doc/info-dir b/doc/info-dir
index 37f0bfa..d27e63c 100644
--- a/doc/info-dir
+++ b/doc/info-dir
@@ -13,7 +13,7 @@ File: dir	Node: Top	This is the top of the INFO tree
 
 * Menu:
 
-Programming Languages
+The Algorithmic Language Scheme
 
 * MIT/GNU Scheme FFI: (mit-scheme-ffi).
                                 Foreign Function Interface
diff --git a/doc/ref-manual/mit-scheme-ref.texi b/doc/ref-manual/mit-scheme-ref.texi
index 7b509f9..39108b0 100644
--- a/doc/ref-manual/mit-scheme-ref.texi
+++ b/doc/ref-manual/mit-scheme-ref.texi
@@ -37,7 +37,7 @@ Documentation License.''
 @comment **** begin CLTL ****
 @comment **** end CLTL ****
 
-@dircategory Programming Languages
+@dircategory The Algorithmic Language Scheme
 @direntry
 * MIT/GNU Scheme Reference: (mit-scheme-ref).
                                 Reference Manual
diff --git a/doc/sos/mit-scheme-sos.texi b/doc/sos/mit-scheme-sos.texi
index 5874a9b..3581e3f 100644
--- a/doc/sos/mit-scheme-sos.texi
+++ b/doc/sos/mit-scheme-sos.texi
@@ -32,7 +32,7 @@ Documentation License.''
 @end quotation
 @end copying
 
-@dircategory Programming Languages
+@dircategory The Algorithmic Language Scheme
 @direntry
 * MIT/GNU Scheme SOS: (mit-scheme-sos).
                                 Scheme Object System
diff --git a/doc/user-manual/mit-scheme-user.texi b/doc/user-manual/mit-scheme-user.texi
index 98a2ac9..f37217a 100644
--- a/doc/user-manual/mit-scheme-user.texi
+++ b/doc/user-manual/mit-scheme-user.texi
@@ -37,7 +37,7 @@ Documentation License.''
 @end quotation
 @end copying
 
-@dircategory Programming Languages
+@dircategory The Algorithmic Language Scheme
 @direntry
 * MIT/GNU Scheme User: (mit-scheme-user).
                                 User's Manual
diff --git a/src/blowfish/mit-scheme-blowfish.texi b/src/blowfish/mit-scheme-blowfish.texi
index a119897..fdde358 100644
--- a/src/blowfish/mit-scheme-blowfish.texi
+++ b/src/blowfish/mit-scheme-blowfish.texi
@@ -24,7 +24,7 @@ Documentation License.''
 @end quotation
 @end copying
 
-@dircategory Programming Languages
+@dircategory The Algorithmic Language Scheme
 @direntry
 * MIT/GNU Scheme Blowfish: (mit-scheme-blowfish).
                                 Blowfish plugin.
diff --git a/src/gdbm/mit-scheme-gdbm.texi b/src/gdbm/mit-scheme-gdbm.texi
index 0e79447..ba20cb7 100644
--- a/src/gdbm/mit-scheme-gdbm.texi
+++ b/src/gdbm/mit-scheme-gdbm.texi
@@ -39,7 +39,7 @@ by the Free Software Foundation.
 @end quotation
 @end copying
 
-@dircategory Programming Languages
+@dircategory The Algorithmic Language Scheme
 @direntry
 * MIT/GNU Scheme GDBM: (mit-scheme-gdbm).
                                 GNU database manager plugin
diff --git a/src/imail/mit-scheme-imail.texi b/src/imail/mit-scheme-imail.texi
index f26f28e..21f2a01 100644
--- a/src/imail/mit-scheme-imail.texi
+++ b/src/imail/mit-scheme-imail.texi
@@ -30,7 +30,7 @@ Documentation License.''
 @end quotation
 @end copying
 
-@dircategory Programming Languages
+@dircategory The Algorithmic Language Scheme
 @direntry
 * MIT/GNU Scheme IMAIL: (mit-scheme-imail).
                                 IMAP/MIME email reader
